exodus_core/analysis/static_analysis.py

```diff
@@ -16,8 +16,8 @@
 import subprocess
 import tempfile
 import time
-import urllib.request
 import zipfile
+import requests
```

Comment:
Please order the imports alphabetically :)

Reply:
I will fix it.